### ISSUE TITLE
Redshift does not work in raspberry pi

### ISSUE THREAD

**Issue by pickfire** [jonls/redshift#257]
Hi, I had tested redshift with different configuration on Raspberry Pi. It seems that there are no changes to the monitor, is this an issue about the video driver?

**Comment by maandree** [comment]
If I remember correctly: Raspberry Pi does not support gamma correction. Thus, it does not support Redshift.

**Comment by pickfire** [comment]
Okay, I think this is out of the topic now. I will close this until a solution is found.

**Comment by mskwon** [comment]
Redshift now works with the experimental Raspberry Pi OpenGL driver.

**Comment by pickfire** [comment]
Nice! Thanks a lot.

**Comment by max-circlefade** [comment]
Hey @mskwon ! Is there any thing I need to tweak to get it to work on Rpi ? Doesn't seem to work on my side, can't initialize the geolocation :/ Thanks

**Comment by Thovthe** [comment]
> Hey @mskwon ! Is there any thing I need to tweak to get it to work on Rpi ? Doesn't seem to work on my side, can't initialize the geolocation :/ Thanks

I recommend setting dawn-time and dusk-time in the config file redshift.conf.  Easier than dealing with location services.  This wont make it work but it makes it not break because of the provider.

**Comment by jimjamz** [comment]
> Redshift now works with the experimental Raspberry Pi OpenGL driver.

Care how to share you got this to work?
I can confirm that the GL driver that Raspbian (now Raspberry Pi OS) includes ***doesn**'t* resolve this issue.  The driver can be selected within `raspi-config`, however upon rebooting the Pi and restarting redshift (which _is_ running), the visual effects (colour temperature, brightness, gamma) are not produced.  I have also tried this by installing other Debian distros on the Pi, where I have confirmed redshift to work on those distros but on other architectures, i386, amd64.  I have included the raspbian repository on those distros to enable the GL KMS driver, but still to no avail.
